app/controllers/franklin_alma_controller.rb

```diff
@@ -142,39 +147,38 @@ def portfolio_details
     public_note_content = public_note.nil? || public_note.empty? ? [] : ["Public Notes: ", public_note]
     authentication_note_content = authentication_note.nil? || authentication_note.empty? ? [] : ["Authentication Notes: ", authentication_note]
 
-    render :json => { "portfolio_details": coverage_content.join("<br/>").html_safe, "notes": (public_note_content + authentication_note_content).join("<br/>").html_safe }
+    render json: { "portfolio_details": coverage_content.join("<br/>").html_safe,
+                   "notes": (public_note_content + authentication_note_content).join("<br/>").html_safe
+    }
   end
 
   def has_holding_info?(api_mms_data, mmsid)
     # check if any holdings have more than one item
-    has_holding_info = api_mms_data['availability'][mmsid]['holdings'].map(&:keys).reduce([], &:+).member?('holding_info') ||
-                       api_mms_data['availability'][mmsid]['holdings'].any? { |hld| hld['total_items'].to_i > 1 || hld['availability'] == 'check_holdings' }
+    api_mms_data['availability'][mmsid]['holdings'].map(&:keys).reduce([], &:+).member?('holding_info') ||
+      api_mms_data['availability'][mmsid]['holdings'].any? { |hld| hld['total_items'].to_i > 1 || hld['availability'] == 'check_holdings' }
   end
 
   def has_portfolio_info?(api_mms_data, mmsid)
-    has_portfolio_info = api_mms_data['availability'][mmsid]['holdings'].map(&:keys).reduce([], &:+).member?('portfolio_pid')
+    api_mms_data['availability'][mmsid]['holdings'].map(&:keys).reduce([], &:+).member?('portfolio_pid')
   end
 
   def single_availability
-    availability_status = {'available' => 'Available',
-                           'check_holdings' => 'Requestable'}
+    availability_status = { 'available' => 'Available',
+                            'check_holdings' => 'Requestable' }
 
     api_instance = BlacklightAlma::BibsApi.instance
     api = api_instance.ezwadl_api[0]
 
     mmsid = params[:mms_id]
-    userid = session['id'].presence || 'GUEST'
-    bibapi = alma_api_class.new()
+    userid = session['id'] || 'GUEST'
+    bibapi = alma_api_class.new
```

Comment:
This looks like it had been protecting against the possibility of a blank (but non-nil) session id. Are we sure this change is safe?

Reply:
no, not sure - this was premature. i have another PR that cleans up session handling so this this sort of ORing isn't intrspersed everywhere, but that's for another day. i'll undo this, thanks!